_pages/testing.md

```diff
@@ -107,6 +107,23 @@ Login.gov collects and verifies personal information during the proofing process
 
 To simulate a failure, enter a social security number that does not start with “900” or “666”, such as “123-45-6789”.
 
+You can simulate failure to contact a proofing vendor by entering a social security number of “000-00-0000”. See next section for more ways to simulate vendor errors.
```

Comment:
I feel like if we could link a specific "next section" it might be clearer? Just in case

```suggestion
You can simulate failure to contact a proofing vendor by entering a social security number of “000-00-0000”. See ["proofing vendor errors"](#proofing-vendor-errors) for more ways to simulate vendor errors.
```